src/components/social.js

```diff
@@ -9,29 +9,57 @@ module.exports = React.createClass({
     var locale = this.context.intl.locale;
     var twitterShareURL = 'https://twitter.com/share?url=' + appURL +'/' + locale + '/&text=' + encodeURIComponent(this.context.intl.formatMessage({id: 'i_donated_to_mozilla'}));
     var facebookShareURL = 'https://www.facebook.com/sharer/sharer.php?u=' + appURL + '/' + locale + '/';
+    var emailSubject = '';
+    var emailBody = '';
```

Comment:
@beccaklam The strings landed.

So now if you just changed these two lines to:

```
    var emailSubject = this.context.intl.formatMessage({id: 'share_email_subject'})};
    var emailBody = this.context.intl.formatMessage({id: 'share_email_body'})};
```

It should just work. However, those strings were added in another patch after you started yours. So those changes are not tracked in your branch. So if you add what I pasted above it won't do what's expected until we merge it to master, or I(or someone else) shows you how to rebase.